rkt/images.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+	"github.com/coreos/rocket/cas"
+	"github.com/coreos/rocket/common/apps"
+	"github.com/coreos/rocket/pkg/keystore"
+)
+
+// findImages uses findImage to attain a list of image hashes using discovery if necessary
+func findImages(al *apps.Apps, ds *cas.Store, ks *keystore.Keystore) error {
+	if err := al.Walk(func(app *apps.App) error {
```

Comment:
For cases like this, please assign then check in a new conditional
```
err := al.Walk ...
if err != nil
```

... oh actually or you can just return err directly